wordpress-mobile/gutenberg-mobile
# Implement list settings

Fixes #1304 

![Simulator Screen Shot - iPhone 11 Pro Max - 2019-11-26 at 12 08 18](https://user-images.githubusercontent.com/651601/69631776-2a339480-1046-11ea-988a-4c8003621503.png)

To test:
 - Start the demo app
 - Add a new list block
 - Select the ordered list type
 - Add some lines to the list
 - Tap on the settings icon for the list
 - Change the properties start and reversed and see if the block updates correctly.

Update release notes:
